app/src/main/java/ar/com/wolox/android/training/utils/UserSession.java

```diff
@@ -23,16 +18,20 @@ public UserSession(SharedPreferencesManager sharedPreferencesManager) {
 
     public void setEmail(String email) {
         mEmail = email;
-        mSharedPreferencesManager.store(USER_EMAIL, email);
+        mSharedPreferencesManager.store(Extras.UserLogin.EMAIL, email);
     }
 
     public String getEmail() {
         // Really, we don't need to query the username because this instance live as long as the
         // application, but we should add a check in case Android decides to kill the application
         // and return to a state where this isn't initialized.
         if (mEmail == null) {
-            mEmail = mSharedPreferencesManager.get(USER_EMAIL, null);
+            mEmail = mSharedPreferencesManager.get(Extras.UserLogin.EMAIL, null);
         }
         return mEmail;
     }
+
+    public boolean isLoged() {
```

Comment:
just a typo Logged . Remember that you can change every reference of this with  shift+F6